### DISCUSSION TITLE
Nextjs Bundle size

### DISCUSSION THREAD

**Discussion by zeroinfinityofficial** [vercel/next.js#17704] [Help]
I create almost a project that have only two more small files than a nextjs blank boilerplate and didn't even use any library at all still the bundle size is huge. I noticed next pass some common.js files and _app.js, _document.js file all of this files seems redundant and bulkier. Also I noticed that none of this file have good compression for example each of them have a lot of white space and comments which I want to remove. Any idea how to reduce next bundle size? I am already using preact with nextjs

**Comment by timneutkens** [comment]
You're probably looking at the development bundles.
